### PR TITLE
Fix broken error handling

### LINKED ISSUE #31: too few arguments when throwing user exception
```
CRITICAL: ErrorException:sprintf(): Too few arguments {\"errFile\":\"\/code\/src\/Keboola\/DbExtractor\/Extractor\/PgSQL.php\",\"errLine\":122,\"errCode\":0,\"```

https://github.com/keboola/db-extractor-pgsql/blob/master/src/Keboola/DbExtractor/Extractor/PgSQL.php#L120

### PR DESCRIPTION
**Pull request by pivnicek**
Fixes #31

### CODE REVIEW

File: tests/Keboola/Extractor/PgsqlTest.php [hunk 1]
```diff
@@ -969,6 +969,23 @@ public function testThousandsOfTablesGetTables(): void
...
+
+        // use just 1 table
+        unset($config['parameters']['tables'][0]);
+        unset($config['parameters']['tables'][1]);
+        unset($config['parameters']['tables'][2]['columns']);
+        unset($config['parameters']['tables'][2]['table']);
+        $config['parameters']['tables'][2]['query'] = "SELECT %s FROM types";
+
+        $this->setExpectedException(UserException::class);
+
```
Comment (Actimel) [inline tests/Keboola/Extractor/PgsqlTest.php]: Exception message will be `undefined index "table"`? Assert also the message please.

Comment (pivnicek) [reply tests/Keboola/Extractor/PgsqlTest.php]: No it won't, because it's not on config rows. but good point, I'll add the assertion, thanks

**Comment by pivnicek** [comment]
ok @Actimel , updated

**Comment by pivnicek** [comment]
I thought about that, but the code `42601` indicates quite specifically a syntax error.
image
This will be changed anyways when retry by code is finished.

**Pull request merged**